doc/developers_guide/developers_guide.md

```diff
@@ -0,0 +1,11 @@
+# Developers Guide
+
+## Building the native image
+
+Current try to build a native image:
```

Comment:
```suggestion
Currently we build a native image using the following command:
```

Reply:
well, it does not work yet :-/ 
I just wanted to note down my last try somewhere